cloud_edition/flexibility_mode/docs/data_file_transfer.rst

```diff
@@ -1,11 +1,20 @@
-Data File Transfer
-==================
-
-| Depending on your project needs, you may have to transfer data files from/to Akeneo Cloud Environment.
-|
-| Akeneo Flexibility can provide dedicated SFTP access to the environments.
-| This SFTP access is independent from the SSH access used by the integrator to deploy custom code.
-|
-| The SFTP access uses the **akeneosftp** user in a chroot.
-|
-| The chroot is accessible by the akeneo user (so by the PIM application too) from the **/data/transfer/pim** directory on the environment.
+File Transfer (SFTP)
+====================
+
+Transfering files from and to your Akeneo Cloud Environment can be done using an SFTP access.
+
+This access can only be granted **upon request**, after a Cloud ticket has been created through the `helpdesk`_.
+
+Please note that this SFTP access is not related to the SSH access (*i.e: different credentials*) and is limited to one particular folder:  **/data/transfer/pim**.
+
+By default, the username is **akeneosftp** and the port in use is **22**. Also note that you can request multiple SFTP accesses.
+
+Permissions
+-----------
+If Akeneo, as an SSH user or as a PIM process creates files in the SFTP sub-directories, permissions have to be set so that SFTP users can rename or delete them.
```

Comment:
```suggestion
If Akeneo (as an SSH user or as a PIM process) creates files in the SFTP sub-directories, permissions have to be set so that SFTP users can rename or delete them.
```